Added Hero Section

ahmedanwar2 opened the pull request:
A Hero section was added which gives the users two choices to either continue reading more on the home page or to go directly into the testfests.

ahmedanwar2 committed:
Added Hero Section

A Hero section was added which gives the users two choices to either continue reading more on the home page or to go directly into the testfests.

TriggeredBanana reviewed (commented):
1. Kan du pulla main sånn at den er oppdatert før du sende PR? Den er foreløpig 2 commits bak main.
2. I CSS fila så har du ein god del duplikater. For eksempel er et par av dei ".button-container" og ":root", men da er enda fleire og såg eg. Me vil helst ha alt som kan vera samla, blir samla sånn at da ikkje blir vanskelig å fiksa/endra på styling seinare.

ahmedanwar2 committed:
Updated styles.css

Removed duplicate and redundant code seen in styles.css while maintaining the functionality seen in the hero section.

ahmedanwar2 committed:
Merge branch 'main' into Hero-section
